doc/news/changes.h

```diff
@@ -139,6 +139,13 @@ inconvenience this causes.
 <h3>General</h3>
 
 <ol>
+ <li> Changed: As MappingQEulerian before, MappingQ1Eulerian has gained
+ a second constructor that reverts the order of the arguments to indicate
```

Comment:
Two not very important style comments:

1. the first clause is is a fragment (should be 'like MappingQEulerian', or 'Similiar to MappingQEulerian', or similar).
2. 'reverses' or 'switches', not 'reverts'